lldb/test/API/commands/settings/TestSettings.py

```diff
@@ -1018,7 +1018,9 @@ def test_settings_api(self):
 
         # Test OptionValueFileSpec and OptionValueFileSpecList
         setting_path = "target.debug-file-search-paths"
-        setting_value = ["/tmp" "/tmp2"]
+        path1 = os.path.join(self.getSourceDir(), "tmp")
+        path2 = os.path.join(self.getSourceDir(), "tmp2")
+        setting_value = [path1, path2]
         self.runCmd("settings set %s %s" % (setting_path, " ".join(setting_value)))
```

Comment:
```suggestion
        self.runCmd("settings set %s '%s' '%s'" % (setting_path, path1, path2))
```

Might as well quote, just in case.